fly/commands/pipeline_overview.go

```diff
@@ -0,0 +1,147 @@
+package commands
+
+import (
+	"os"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/fly/commands/internal/displayhelpers"
+	"github.com/concourse/concourse/fly/rc"
+	"github.com/concourse/concourse/fly/ui"
+	"github.com/fatih/color"
+)
+
+type PipelineOverviewCommand struct {
+	Pipeline string `short:"p" long:"pipeline" required:"true" description:"Get resources in this pipeline"`
+	Json     bool   `long:"json" description:"Print command result as JSON"`
+}
+
+func (command *PipelineOverviewCommand) Execute(args []string) error {
+	pipelineName := command.Pipeline
+
+	target, err := rc.LoadTarget(Fly.Target, Fly.Verbose)
+	if err != nil {
+		return err
+	}
+
+	err = target.Validate()
+	if err != nil {
+		return err
+	}
+
+	var headers []string
+	var resources []atc.Resource
+
+	resources, err = target.Team().ListResources(pipelineName)
+	if err != nil {
+		return err
+	}
+
+	if command.Json {
+		err = displayhelpers.JsonPrint(resources)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	headers = []string{"name", "type", "pinned"}
```

Comment:
```suggestion
	headers = []string{"resource name", "type", "pinned"}
```

What do you think about explicitly stating that the following data is for resources and same for the jobs table? When I ran the command it took me a couple seconds to figure out "is this first table jobs or is the next one jobs..."

Reply:
You got it, I'll make that update, that's a good point